README.md

```diff
@@ -69,15 +59,8 @@ I would like to receive a text such as "Thank you! Your order was placed and wil
 * Finally submit a pull request before Monday at 9am with your solution or partial solution.  However much or little amount of code you wrote please please please submit a pull request before Monday at 9am
 
 
-In code review we'll be hoping to see:
-
-* All tests passing
-* High [Test coverage](https://github.com/makersacademy/course/blob/master/pills/test_coverage.md) (>95% is good)
-* The code is elegant: every class has a clear responsibility, methods are short etc.
-
-Reviewers will potentially be using this [code review rubric](docs/review.md).  Referring to this rubric in advance will make the challenge somewhat easier.  You should be the judge of how much challenge you want this at this moment.
-
-Notes on Test Coverage
-------------------
+Progress 
+--------
+I have implemented the first three of the four user stories.
 
-You can see your [test coverage](https://github.com/makersacademy/course/blob/master/pills/test_coverage.md) when you run your tests.
+One particular issue was how to ensure that there were always two digits after the decimal place when displaying the price. I ended up adding a string of '0' to pass the tests, which works for my menu but not if the price in pence is not multiples of ten. I tried looking at BigDecimal but it didn't give me the output I wanted. 
```

Comment:
This shows a way of doing string formatting in ruby: https://www.rubyguides.com/2012/01/ruby-string-formatting/